pom.xml

```diff
@@ -121,13 +129,13 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-support</artifactId>
-            <version>2.10</version>
+            <version>${workflow-support-plugin.version}</version>
```

Comment:
🐜 Again, this actually makes it MORE work to read and interact with the pom because now the version I want to change is separated from the usage.  

This would ONLY make sense if you had 3+ usages, or the usages were not *right next* to each other.